### DISCUSSION TITLE
List of good or Bad Feedback about Dotmim.Sync in your projects / products

### DISCUSSION THREAD

**Discussion by Soruk** [Mimetis/Dotmim.Sync#985] [Q&A]
Hi

Do you can share your feedback about of using Dotmim.Sync framework in your projects / products?
Please provide the short description in which way do you use "dms" and what your project do?
It will help to create a list of projects where "dms" is used.

Thanks.

**Comment by SvdSinner** [comment]
We will be using it in our new system we are developing to pull production data (one-way with all PII and sensitive date excluded) into our QAS, TST, INT, and DEV environments.

In testing, it has proven to be far faster and more resilient than our old system of having SQL SSIS packages doing the nightly ETL and the nightmare of trying to keep them all updated and working across new versions and schema changes.
